src/CattyTests/Parser & Serialization/CatrobatLanguage0.991/XMLSerializerTests/Unit Tests/XMLSerializerTests.swift

```diff
@@ -145,4 +145,19 @@ final class XMLSerializerTests: XMLAbstractTest {
         let equal = self.isXMLElement(xmlElement: xmlElement, equalToXMLElementForXPath: xmlElementPath, inProjectForXML: "ValidProjectAllBricks0991")
         XCTAssertTrue(equal, "XMLElement invalid!")
     }
+
+    func testSetPenColorBrick() {
+        let project = self.getProjectForXML(xmlFile: "ValidProjectAllBricks0991")
+
+        let brick = ((project.objectList.object(at: 0) as! SpriteObject).scriptList.object(at: 0) as! Script).brickList.object(at: 45) as! SetPenColorBrick
+        let xmlElementPath = "//program/objectList/object[1]/scriptList/script[1]/brickList/brick[46]"
+
+        guard let xmlElement = brick.xmlElement(with: CBXMLSerializerContext()) else {
```

Comment:
Please rebase from the current version of develop and fix this test.

Reply:
The initializer of `CBXMLSerializerContext` has changed, that's probably the reason why your build is not compiling.